xbmc/cores/VideoPlayer/DVDDemuxers/DVDDemux.h

```diff
@@ -243,29 +243,29 @@ class CDVDDemux
   /*
    * Reset the entire demuxer (same result as closing and opening it)
    */
-  virtual void Reset() = 0;
+  virtual void Reset() { }
 
   /*
    * Aborts any internal reading that might be stalling main thread
    * NOTICE - this can be called from another thread
    */
-  virtual void Abort() = 0;
+  virtual void Abort() { }
 
   /*
    * Flush the demuxer, if any data is kept in buffers, this should be freed now
    */
-  virtual void Flush() = 0;
+  virtual void Flush() { }
 
   /*
    * Read a packet, returns NULL on error
    *
    */
-  virtual DemuxPacket* Read() = 0;
```

Comment:
right, I mean those are required by VP. if overridden they can be used to change behaviour.  VP uses those methods to control demuxers and get info from demuxers. SetSpeed does not belong to this category. it was implemented because the demuxer wanted info from VP.